src/devices/Ad7193/Ad7193.cs

```diff
@@ -0,0 +1,855 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Device.Spi;
+using System.Diagnostics;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Ad7193.Metadata;
+using global::Iot.Units;
```

Comment:
is global:: necessary?

Reply:
Strange, when I removed it in the past it caused a compilation error, but now it looks fine. I removed it.